scripts/link-validator.conf

```diff
@@ -0,0 +1,71 @@
+// config for https://github.com/ennru/site-link-validator/
+site-link-validator {
+  root-dir = "/Users/enno/dev/akka/akka-docs/target/paradox/site/main/"
+  # relative to `root-dir`
+  start-file = "index.html"
+
+  # Resolves URLs with the given prefix as local files instead
+  link-mappings = [
+    {
+      prefix = "https://doc.akka.io/docs/akka/current/"
+      replace = ""
+    }
+    # ScalaDoc from unidoc
+    {
+      prefix = "https://doc.akka.io/api/akka/2.6/"
+      replace = "/../../../../../target/scala-2.13/unidoc/"
+    }
+    {
+      prefix = "https://doc.akka.io/api/akka/current/"
+      replace = "/../../../../../target/scala-2.13/unidoc/"
+    }
+    {
+      prefix = "https://doc.akka.io/api/akka/snapshot/"
+      replace = "/../../../../../target/scala-2.13/unidoc/"
+    }
+    # Java APIs from genjavadoc
+    {
+      prefix = "https://doc.akka.io/japi/akka/2.6/"
+      replace = "/../../../../../target/javaunidoc/"
+    }
+    {
+      prefix = "https://doc.akka.io/japi/akka/current/"
+      replace = "/../../../../../target/javaunidoc/"
+    }
+    {
+      prefix = "https://doc.akka.io/japi/akka/snapshot/"
+      replace = "/../../../../../target/javaunidoc/"
+    }
+  ]
+
+  ignore-missing-local-files-regex = ""
+  // e.g. "^api/alpakka/snapshot/akka/stream/alpakka/googlecloud/storage/impl/Formats.*"
+
+  ignore-prefixes = [
+    # Fails after a number of requests with "403 Forbidden"
+    "https://javadoc.io/static/"
+    # GitHub will block with "429 Too Many Requests"
+    "https://github.com/"
+    "https://www.scala-lang.org/api/2.13.7/scala/runtime/AbstractFunction1.html"
+    "https://www.scala-lang.org/api/2.13.7/scala/runtime/AbstractFunction2.html"
+    "https://www.scala-lang.org/api/2.13.7/scala/runtime/AbstractFunction3.html"
+    "https://www.scala-lang.org/api/2.13.7/scala/runtime/AbstractPartialFunction.html"
+  ]
+
+  non-https-whitelist = [
+    "http://cidrdb.org/cidr2015/Papers/CIDR15_Paper16.pdf"
+    "http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.41.7628"
+    "http://spray.io"
+    "http://www.bailis.org/blog/doing-redundant-work-to-speed-up-distributed-queries/"
+    "http://www.cs.columbia.edu/~nahum/w6998/papers/sosp87-timing-wheels.pdf"
+    # document not available anymore
+    "http://www.jaist.ac.jp/~defago/files/pdf/IS_RR_2004_010.pdf"
+    # Sigar site has been removed
+    "http://www.hyperic.com/products/sigar"
```

Comment:
We might want to follow up on these. The links are effectively broken.

Reply:
https://github.com/akka/akka/issues/31089